app/home/homeController.js

```diff
@@ -8,7 +8,9 @@
 
     function controllerFunction (cityService) {
           var vm = this;
-          vm.city = cityService.city;
+          vm.cityService = cityService;
+          // Curious as to why you need the whole object. Commented out in forecastController.
```

Comment:
Hmm, interesting why it wouldn't work. You should only need to attach primitives to an object when you still want to access and update it within a directive that inherits the parent scope, like `ng-if`...but I don't see any directives on your template. Oh well.